spec/integration/metric_spec.rb

```diff
@@ -135,6 +135,7 @@
           id1 = SecureRandom.uuid
           id2 = SecureRandom.uuid
           id3 = SecureRandom.uuid
+          id4 = SecureRandom.uuid
 
           ids = [id1, id2, id3]
           bindings = { id1: id1, id2: id2, id3: id3 }
```

Comment:
Could you please add id4 to this hash (`bindings`)?
It will help VCR do its replace-template-magic-thing.